polaris.shopify.com/content/foundations/foundations/about-polaris/index.md

```diff
@@ -0,0 +1,75 @@
+
+### Primitives
+
+Primitives are the building blocks of the Polaris design system. They are the smallest units of the system that represent codified design decisions and reusable visual elements. The Polaris design system contains primitive tokens and primitive components that can be combined together to create admin user interfaces.
+
+### Tokens
+
+Design tokens represent reusable design decisions such as color, spacing, and typography. These tokens are used throughout the Polaris design system and Shopify admin to unify the merchant experience across the entire platform.
+
+Polaris design tokens are packaged up as the open source [@Shopify/polaris-tokens](https://github.com/Shopify/polaris/tree/main/polaris-tokens) library
+
+### Components
+
+Polaris components are the smallest reusable user interface elements for supporting a variety of admin merchant experiences. Polaris components are one of our design system primitives, and are often referred to as primitive components.
+
+These primitive components are highly composable and can be assembled together to create a variety of patterns and user experiences within the admin.
+
+Polaris web components are packaged as the open source [@Shopify/polaris](https://github.com/Shopify/polaris/tree/main/polaris-react) React UI library. The Polaris React library is used in the Shopify admin web app as well as within embedded web views in the Shopify admin iOS and Android apps.
+
+Polaris React Native components are also used in the Shopify admin iOS and Android apps. The Polaris React Native library is only available to Shopify employees building with React Native.
+
+### Compositions
+
+Polaris tokens and components can be assembled together to solve any number of merchant needs in the admin. Compositions_ _are the result of design tokens and/or primitives being put together to create a specific user experience. 
+
+Many of our primitive components are built with Polaris design tokens, and many of our design patterns can be composed using primitive components. 
+
+Some compositions can be packaged and distributed via component libraries. 
+
+### Design patterns
+
+Design patterns are best practice design guidance. They address situations that merchants commonly face, including:
+
+* Tasks, like searching and filtering
+* Situations, like errors and loading
+* UI characteristics, like feedback
+* Architecture, like navigation and page layout
+
+These patterns center on the merchant, and how we build experiences to meet their needs.
+
+Design patterns often use one or more primitive components and adapt them to the context. Some patterns can link to compositions and will contain code examples where possible.
+
+For a deeper understanding of design patterns, read our guide to [admin design patterns](/foundations/patterns/design-patterns).
```

Comment:
Don't merge this until the patterns page gets merged in!

https://github.com/Shopify/polaris/pull/6856